src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public abstract class Figures implements Draw {
+    private final String figure;
+    private final String color;
+
+    public Figures(String figure, String color) {
+        this.figure = figure;
+        this.color = color;
+    }
+
+    public String getFigure() {
+        return figure;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public static void main(String[] args) {
+
+        Figures[] array = new Figures[6];

```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#all-magic-numbers-in-your-code-should-be-constants